public/request/admin.php

```diff
@@ -82,23 +82,3 @@
 
     return back()->with('success', __('legacy.success.ok'));
 }
-
-if ($action === 'aotw') {
-    $aotwAchID = requestInputSanitized('a', 0, 'integer');
-    $aotwForumID = requestInputSanitized('f', 0, 'integer');
-    $aotwStartAt = requestInputSanitized('s', null, 'string');
-
-    $query = "UPDATE StaticData SET
-        Event_AOTW_AchievementID='$aotwAchID',
-        Event_AOTW_ForumID='$aotwForumID',
-        Event_AOTW_StartAt='$aotwStartAt'";
```

Comment:
Should we have a migration to drop these columns? Or just let them stagnate? I don't think the table will be going away any time soon.

Reply:
I think we can probably drop around half of that table's columns. I think we should do it, but let's push that off to a subsequent PR.